pkg/signutil/cosignutil.go

```diff
@@ -46,6 +46,7 @@ func SignCosign(rawRef string, keyRef string) error {
 		cosignCmd.Env = append(cosignCmd.Env, "COSIGN_EXPERIMENTAL=true")
 	}
 
+	cosignCmd.Args = append(cosignCmd.Args, "--yes")
```

Comment:
Hi @ningziwen , Is it relevant to expose the prompt feature as nerdctl options ? And make it silent within integration tests .

Reply:
This is not to expose --yes, but to add --yes to make it silent by default. So it can be silent within integration tests and in the downstream software that is comsuming nerdctl.

The alternative option is to expose --cosign-yes in nerdctl. But I feel as Cosign is treated as the dependency of nerdctl feature, nerdctl can be kind of opininated to say yes to it, and make users have the expection of saying yes to this. (we can also add some documentation if needed)

Reply:
LGTM but please introduce some documentation

Reply:
FYI prompt  can be disablied by exporting `
COSIGN_PASSWORD` 
[source](https://github.com/containerd/nerdctl/blob/main/docs/cosign.md )
```
# Export your COSIGN_PASSWORD to prevent CLI prompting
$ export COSIGN_PASSWORD=$COSIGN_PASSWORD
```

Reply:
I think COSIGN_PASSWORD only disables the prompt of inputting cosign key passphrase. The prompt in this PR is only for private statement, which can't be disabled by COSIGN_PASSWORD.

```
	PrivacyStatement = `
        Note that there may be personally identifiable information associated with this signed artifact.
        This may include the email address associated with the account with which you authenticate.
        This information will be used for signing this artifact and will be stored in public transparency logs and cannot be removed later.`
	PrivacyStatementConfirmation = "        By typing 'y', you attest that you grant (or have permission to grant) and agree to have this information stored permanently in transparency logs."
```

`--yes` is also specifically for privacy statement.

Reply:
Added documentation.

Reply:
From a privacy point of view, I think it is better to make it explicit for nerdctl users imo. We should make our users opt-in.

Reply:
agree regarding `privacy point of view` purpose, but I think the idea here is to unblock the upgrade to `2.0.0`, so let's accept it as default value, and make new PR to make it explicit for nerdctl users